os/net/ipv6/ip64-addr.c

```diff
@@ -34,9 +34,12 @@
 #include <stdio.h>
 #include <string.h>
 
-#define printf(...)
+/* Log configuration */
+#include "sys/log.h"
+#define LOG_MODULE "IPv6"
+#define LOG_LEVEL LOG_LEVEL_NONE
 
-static uip_ip6addr_t ip64_prefix = {{ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0xff, 0xff, 0, 0, 0, 0}};
+static uip_ip6addr_t ip64_prefix = {{ 0, 0x64, 0xff, 0x9b, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}};
```

Comment:
I think, we may need this change in a separate commit, explaining in its commit log why the prefix is modified. It seems this change correspond to https://tools.ietf.org/html/rfc6052#section-2.1?

Reply:
Yes, this is changed since the normally used NAT64 prefix seems to be 64:ff9b::/96. I can split it into a separate commit if that makes sense - or document it in  the source code.

Reply:
Thanks! :+1: